src/screens/profile-screen/edit-profile-screen.tsx

```diff
@@ -23,9 +55,26 @@ export const EditProfileScreen: React.FC<EditProfileProps> = () => {
         console.log('console from submit');
     };
 
+    if (isLoading || isFetching) {
+        return <Text>Loading</Text>;
+    }
+
+    if (isError) {
+        return <Text>{error}</Text>;
+    }
+
     return (
         <>
-            <ImageUploader imageProp="https://wallpaperaccess.com/full/317501.jpg" />
+            {data && isSuccess && <ImageUploader setImageState={setImageState} imageProp={data} />}
```

Comment:
Should handle the fallback image internally to ImageUploader, instead of like this. That way, it still displays that image component if the data is undefined.

Reply:
Also, wrap the screens in a <Box/> or some other component, usually an empty component like this is less reliable with styling